src/components/FiltersAndSearch/SearchInput/SearchInput.vue

```diff
@@ -98,40 +98,44 @@ export default {
 
 <style lang='scss' scoped>
 .pb-search-input-container {
-  width: 100%;
-  position: relative;
-
+  display: flex;
+  flex-direction: row;
```

Comment:
o default de um display flex já seria um row

Reply:
Sim, to ligado. Mas se eu coloquei muito provavelmente ele estava pegando column por algum motivo.